templates/install-wp-tests.sh

```diff
@@ -147,6 +147,7 @@ install_db() {
 	fi
 
 	# create database
+	mysqladmin drop $DB_NAME -f --user="$DB_USER" --password="$DB_PASS"$EXTRA > /dev/null 2>&1
```

Comment:
I don't think this is the right solution! Dropping the database with no warning is going to cause problems for someone eventually.

Reply:
I had the same thought but isn't this script intended to be used to initialize the environment? I'm more than happy to make changes, including asking the user to confirm the deletion of the database prior to deleting it. Just let me know what seems reasonable.

Reply:
>asking the user to confirm the deletion of the database prior to deleting it

This seems reasonable to me but the best person to confirm is @schlessera